core/src/main/java/io/camunda/migrator/RuntimeMigrator.java

```diff
@@ -162,11 +162,10 @@ protected void fetchProcessInstancesToMigrate(Consumer<IdKeyDbModel> storeMappin
 
     } else {
       LOGGER.debug("Fetching latest start date of process instances");
-      Date latestStartDate = callApi(() -> idKeyMapper.findLatestStartDateByType(TYPE.RUNTIME_PROCESS_INSTANCE));
-      LOGGER.debug("Latest start date: {}", latestStartDate);
+      IdKeyDbModel latestProcessInstance = callApi(() -> idKeyMapper.findByTypeOrderedByStartDateDesc(TYPE.RUNTIME_PROCESS_INSTANCE));
+      LOGGER.debug("Latest process instance [{}]", latestProcessInstance);
```

Comment:
On the first run this message is `Latest process instance [null]` but that's expected.

Reply:
Yes. That's was the case already before. This basically means all process instances will be queried.